p/scripts/main.js

```diff
@@ -1542,6 +1542,7 @@ function init_normal() {
 }
 
 function init_beforeDOM() {
+	document.scrollingElement.scrollTop = 0;
```

Comment:
This actually works? Interesting. :-)

Reply:
Apparently yes, but more tests welcome :-)